.travis.yml

```diff
@@ -0,0 +1,4 @@
+sudo: false
+language: node_js
+node_js:
+- 8
```

Comment:
\* kuck kuck *

Reply:
What's the problem with Node 8? It's the last LTS!

Reply:
Haha, no, it's the no new line, which you see a red icon for in the files tab, but not here in the inline comments.